src/game/common/compression/compressionmanager.h

```diff
@@ -35,12 +35,22 @@ enum CompressionType : int32_t
     COMPRESSION_COUNT,
 };
 
+struct ComprHeader
+{
+    char fourcc[4];
+    uint32_t uncomp_size;
+};
+
 class CompressionManager
 {
 public:
     static bool Is_Data_Compressed(const void *data, int size);
+    static CompressionType Get_Prefered_Compression();
     static CompressionType Get_Compression_Type(const void *data, int size);
+    static int Get_Max_Compressed_Size(int size, CompressionType type);
+    static const char *Get_Compression_FourCC(CompressionType type);
```

Comment:
Are these new functions as per original or are these custom for Thyme?

Reply:
Apart from `Get_Compression_FourCC` all of these new functions exist in the original. Atleast that's the information @tomsons26  provided me with. I could probably get rid of `Get_Compression_FourCC` but i find it easier to read if there's a single place for that, instead of putting it everywhere

Reply:
Could you perhaps add a comment for `Get_Compression_FourCC` that it is Thyme specific?